packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -86,7 +86,7 @@ double getHintOpacity(WidgetTester tester) {
     find.ancestor(
       of: find.text('hint'),
       matching: find.byType(FadeTransition),
-    ).last
+    ).first
```

Comment:
did you check if the bug was with the test or with the framework?

Reply:
I'm not super sure what you meant. I checked here by dumping the tree and checking that the test author meant to check the closest FadeTransition to the text which is the first element up the ancestry chain rather than the (coincidentally) last item in the normal full tree walk (which is the bug).

Reply:
great, that's what i meant.